integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkUtilIT.java

```diff
@@ -108,6 +108,26 @@ protected List<String> getQueryData(String querySql) {
         .collect(Collectors.toList());
   }
 
+  // columns data are joined by ','
+  protected List<String> getTableMetadata(String getTableMetadataSql) {
+    return sql(getTableMetadataSql).stream()
```

Comment:
please simplify the code, change to:

// columns data are joined by ','
protected List<String> getTableMetadata(String getTableMetadataSql) {
   return getQueryData(getTableMetadataSql);
}

Reply:
updated.